core/src/core/block.rs

```diff
@@ -1065,7 +1065,7 @@ mod test {
 		};
 		assert_eq!(
 			vec.len(),
-			target_len,
+			target_len
```

Comment:
I think our "style guide" has trailing commas in lists of args like this.
Is this happening in your editor? Or rustfmt?

Reply:
Found out that my rustc was not updated and did not accept these trailing commas. Reverted.